debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/structureformat/extensions/J9ModuleStructureFormatter.java

```diff
@@ -50,21 +53,20 @@ public FormatWalkResult postFormat(String type, long address,
 	{
 		if (type.equalsIgnoreCase("j9module") && address != 0) {
 			J9ModulePointer modulePtr = J9ModulePointer.cast(address);
-
+			String moduleAddress = modulePtr.getHexAddress();
 			try {
 				out.println("Module name: " + J9ObjectHelper.stringValue(modulePtr.moduleName()));
 			} catch (CorruptDataException e) {
 				// Do nothing
 				return FormatWalkResult.KEEP_WALKING;
 			}
-			out.println("To display all j9packages exported by the module, use !dumpmoduleexports "
-					+ modulePtr.getHexAddress());
-			out.println("To display all modules that the target module reads, use !dumpmodulereads "
-					+ modulePtr.getHexAddress());
+			out.println("To display all j9packages exported by the module, use !dumpmoduleexports " + moduleAddress);
+			out.println("To display all modules that the target module reads, use !dumpmodulereads " + moduleAddress);
+			out.println("To find all modules that read the module, use !findallreads " + moduleAddress);
```

Comment:
"that read the" --> "that read this"

Reply:
No, all the mentions of 'the module' read better as 'a module'; there isn't a 'this' in context that's meaningful to the human reading these messages.